gencapdefs.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python3
+
+"""
+Updates the capability definitions at irc/caps/defs.go
+
+To add a capability, add it to the CAPDEFS list below,
+then run `make capdefs` from the project root.
+"""
+
+import io
+import subprocess
+import sys
+from collections import namedtuple
+
+CapDef = namedtuple("CapDef", ['identifier', 'name', 'url', 'standard'])
+
+CAPDEFS = [
+    CapDef(
+        identifier="LabelTagName",
```

Comment:
LabelTagName is just a var that refers to the string `"draft/label"`, since that's likely to change in the future and it's referred to around the code in various places.

Especially with `caps.XYZ.Name()` being used, we can just do refer to this as `Label` and then the calls in the code will just be `caps.Label.Name()` instead :)